patches.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
```

Comment:
Are we required to use `/bin/sh`?  I've had good experience with `set -e` and `set -o pipefail` in bash.

Reply:
I have no experience with writing bash scripts, so I chose `/bin/sh` because it seemed to be common across most platforms.